src/ingest-pipeline/misc/tools/split_and_create.py

```diff
@@ -123,13 +130,7 @@ def create_new_uuid(row, source_entity, entity_factory, dryrun=False):
             info_txt_root = f"Upload {source_entity.prop_dct['hubmap_id']}"
     assert info_txt_root is not None, "Expected a Dataset or an Upload"
     info_txt = info_txt_root + " : " + rec_identifier
-    try:
-        type_info = entity_factory.type_client.getAssayType(canonical_assay_type)
-    except Exception:
-        print(f"tried {orig_assay_type}, canoncal version {canonical_assay_type}")
-        print(f"options are {list(entity_factory.type_client.iterAssayNames())}")
-        type_info = entity_factory.type_client.getAssayType(orig_assay_type)
-    contains_human_genetic_sequences = type_info.contains_pii
+    contains_human_genetic_sequences = primary_entity.get("contains-pii")
```

Comment:
Is this taking care of the refactor that Zhou mentioned to get rid of the type_client?

Reply:
Yes, it probably needs more work around that in other parts of the code for different functionality.